src/main/java/seedu/address/model/modelGeneric/AddressBookGeneric.java

```diff
@@ -106,8 +106,8 @@ public K get(ID objID) {
         return objects.get(objID);
     }
 
-    public K remove(ID id) {
-        return objects.remove(id);
+    public void remove(ID id) {
```

Comment:
I think we should't make remove a void method as old implementation doesn't affect removing an object and it is more similar to remove method in java api which returns removed object.

Reply:
nvm I think it's ok our remove/delete method declared in Model also void